loader.qml

```diff
@@ -1,109 +1,313 @@
-import QtQuick 2.15
 import QtGraphicalEffects 1.12
-import QtQuick.Layouts 1.2
-import QtQuick.Controls 1.4
+import QtQuick 2.15
+import QtQuick.Controls 2.15
 import QtQuick.Dialogs 1.1
+import QtQuick.Layouts 1.2
 
 ApplicationWindow {
-    id: root
-    property bool displayAmbient: false
-    property bool round: true
+    id: approot
+    property bool displayAmbient: ambientCheckBox.checked
     property var testface: Qt.application.arguments[1]
```

Comment:
And testFace.
Also what does testFace mean? it's unclear from the name. I think watchfacePath would be nicer, then it'd be clearer that it's a path that points to a watchface.

Reply:
I renamed it from `testface` to `nameOfWatchfaceToBeTested`.  Seems a bit long, but more descriptive, I suppose.